### PR TITLE
[PPML] Allow users to turn off dcap attestation when building image

### PR DESCRIPTION
**Pull request by Uxito-Ada**
## Description

Add a flag in build console, which allow users to turn off dcap.

### 1. Why the change?

Many customers want a non-attestation for demo.

### 2. User API changes

None.

### 3. Summary of the change 

Allow users to turn off dcap attestation when building image.

### 4. How to test?
- [ ] N/A
- [ ] Unit test
- [ ] Application test
- [x] Document test
- [ ] ...

### CODE REVIEW

**Comment by Uxito-Ada** [comment]
Passed local image build test.

File: ppml/trusted-big-data-ml/python/docker-gramine/bigdl-gramine/Dockerfile [hunk 1]
```diff
@@ -6,14 +6,15 @@ FROM $BASE_IMAGE_NAME:$BASE_IMAGE_TAG as temp
 
 ARG SGX_MEM_SIZE
 ARG SGX_LOG_LEVEL
+ARG ATTESTATION
```
Comment (glorysdj) [inline ppml/trusted-big-data-ml/python/docker-gramine/bigdl-gramine/Dockerfile:9]: Attestation change to ENABLE_DCAP_ATTESTATION

**Review by qiyuangong — APPROVED** [review]
LGTM

File: ppml/trusted-big-data-ml/python/docker-gramine/bigdl-gramine/Dockerfile (hunk 1 above)
Comment (Uxito-Ada) [reply ppml/trusted-big-data-ml/python/docker-gramine/bigdl-gramine/Dockerfile]: fixed.

**Pull request merged**